src/Components/Navbar/index.jsx

```diff
@@ -0,0 +1,23 @@
+export function Navbar({ setCurrentPage, pages }) {
+  const items = [
+    { id: 1, name: "Home", onClick: () => setCurrentPage(pages.home) },
+    { id: 2, name: "Profile", onClick: () => setCurrentPage(pages.profile) },
+    { id: 5, name: "Login" },
+  ];
+  return (
+    <nav className="bg-slate-100 text-slate-900 p-4 ">
+      <ul className="flex gap-4 justify-evenly">
```

Comment:
justify-evenly does not work well for nav menus. Use justify-start or justify-end instead.
```suggestion
      <ul className="max-w-screen-md mx-auto flex gap-4 justify-start">
```